pom.xml

```diff
@@ -67,11 +67,20 @@
         </developer>
     </developers>
 
+    <repositories>
+        <repository>
+            <id>repository.spring.milestone</id>
+            <name>Spring Milestone Repository</name>
+            <url>https://repo.spring.io/milestone</url>
+        </repository>
+    </repositories>
+
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hibernate.version>5.6.10.Final</hibernate.version>
-        <spring.version>5.3.22</spring.version>
-        <liquibase.version>0-SNAPSHOT</liquibase.version>
+        <hibernate.version>6.1.3.Final</hibernate.version>
+        <spring.version>6.0.0-M6</spring.version>
```

Comment:
Spring version can be bumped to the latest 6.0.2 
https://github.com/spring-projects/spring-framework/releases/tag/v6.0.2